python/reg_interface/scripts/repeated_reg_read.py

```diff
@@ -0,0 +1,139 @@
+            transaction_cnt_before
+         ))
+
+    node = getNode(args.reg_name)
+
+    register_values = {}
+    
+    for i in range(0,int(args.nreads)):
+        value = readAddress(node.real_address)
+
+        if value in register_values.keys():
+            register_values[value] += 1
+        else:
+            register_values[value] = 1
+        
+        time.sleep(float(args.sleeptime)/float(1000000))
+
+    crc_error_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.CRC_ERROR_CNT").real_address)
+    packet_error_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.PACKET_ERROR_CNT").real_address)
+    bitstuffing_error_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.BITSTUFFING_ERROR_CNT").real_address)
+    timeout_error_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.TIMEOUT_ERROR_CNT").real_address)
+    axi_strobe_error_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.AXI_STROBE_ERROR_CNT").real_address)
+    transaction_cnt_after = readAddress(getNode("GEM_AMC.SLOW_CONTROL.VFAT3.TRANSACTION_CNT").real_address)
+
+    crc_error_cnt_delta = str(int(crc_error_cnt_after,16) - int(crc_error_cnt_before,16))
+    packet_error_cnt_delta = str(int(packet_error_cnt_after,16) - int(packet_error_cnt_before,16))
+    bitstuffing_error_cnt_delta = str(int(bitstuffing_error_cnt_after,16) - int(bitstuffing_error_cnt_before,16))
+    timeout_error_cnt_delta = str(int(timeout_error_cnt_after,16) - int(timeout_error_cnt_before,16))
+    axi_strobe_error_cnt_delta = str(int(axi_strobe_error_cnt_after,16) - int(axi_strobe_error_cnt_before,16))
+    transaction_cnt_delta = str(int(transaction_cnt_after,16) - int(transaction_cnt_before,16))
+
+    f.write("| after | {0}  | {1} | {2} | {3} | {4} | {5} |\n".format(
+            crc_error_cnt_after,
+            packet_error_cnt_after,
+            bitstuffing_error_cnt_after,
+            timeout_error_cnt_after,
+            axi_strobe_error_cnt_after,
+            transaction_cnt_after
+         ))
+    
+    print("| after | {0}  | {1} | {2} | {3} | {4} | {5} |".format(
+            crc_error_cnt_after,
+            packet_error_cnt_after,
+            bitstuffing_error_cnt_after,
+            timeout_error_cnt_after,
+            axi_strobe_error_cnt_after,
+            transaction_cnt_after
+         ))
+
+    f.write("| delta | {0}  | {1} | {2} | {3} | {4} | {5} |\n".format(
+            crc_error_cnt_delta,
+            packet_error_cnt_delta,
+            bitstuffing_error_cnt_delta,
+            timeout_error_cnt_delta,
+            axi_strobe_error_cnt_delta,
+            transaction_cnt_delta
+         ))
+    
+    print("| delta | {0}  | {1} | {2} | {3} | {4} | {5} |".format(
+            crc_error_cnt_delta,
+            packet_error_cnt_delta,
+            bitstuffing_error_cnt_delta,
+            timeout_error_cnt_delta,
+            axi_strobe_error_cnt_delta,
+            transaction_cnt_delta
+         ))
```

Comment:
For all of your `write`/`print` calls, you could reduce the duplication you have here by just creating a string of the thing you're going to `write`/`print`, and then just calling, e.g., 
```python
outstring = "blah".format(values)
f.write(outstring)
print(outstring)
```